force-app/main/default/lwc/errorSettings/errorSettings.html

```diff
@@ -21,6 +21,73 @@
                 disabled={affordancesDisabled}
             ></c-settings-row-input>
 
+            <!-- Send Error Notigication Setting -->
+            <c-settings-row-input
+                class="slds-size_3-of-3"
+                title={labelReference.sendErrorsSettingTitle}
+                description={labelReference.stgHelpErrorNotifyOn}
+                type="toggle"
+                input-id={inputAttributeReference.sendErrorNotificationsToggleId}
+                value={errorSettingsVModel.sendErrorNotifications}
+                checked={errorSettingsVModel.sendErrorNotifications}
+                onsettingsinputchange={handleSendErrorNotificationsChange}
+                disabled={affordancesDisabled}
+            ></c-settings-row-input>
+
+            <template if:true={showErrorNotificationRecipients}>
+                <!-- Send Error Notification Category -->
+                <c-settings-row-input
+                    class="slds-size_3-of-3 slds-var-p-left_x-large"
+                    title={labelReference.errorNotificationRecipientsTitle}
+                    description={labelReference.errorNotificationRecipientsDescription}
+                    type="combobox"
+                    input-id={inputAttributeReference.errorNotificationRecipientsComboboxId}
+                    value={errorSettingsVModel.errorNotificationsRecipientCategory.value}
+                    placeholder={labelReference.comboboxPlaceholderText}
+                    options={errorSettingsVModel.errorNotificationsRecipientCategory.options}
+                    onsettingsinputchange={handleErrorNotificationRecipientCategoryChange}
+                    disabled={affordancesDisabled}
+                ></c-settings-row-input>
+                <template if:true={showErrorNotificationRecipientChatterGroupLookup}>
+                    <div class="slds-col slds-size_2-of-3"></div>
+                    <div class="slds-col slds-size_1-of-3">
+                        <div class="slds-grid slds-grid_align-end slds-var-p-right_x-large slds-var-p-bottom_x-large">
+                            <div class="slds-col">
+                                <c-single-lookup
```

Comment:
The top padding is too large imo.

Reply:
Agreed on this, but to change that I would need to change the settings input component which introduces too much risk IMHO.

Reply:
right agreed, maybe not a good idea to include padding in single component but i guess we will get it fixed someday.